packages/jazz-tools/src/tools/coValues/interfaces.ts

```diff
@@ -710,6 +709,29 @@ export async function exportCoValue<
   return contentPieces;
 }
 
+export function exportCoValueFromSubscription(
+  subscription: SubscriptionScope<CoValue>,
+): ExportedCoValue {
+  const valuesExported = new Set<string>();
+  const contentPieces: CojsonInternalTypes.NewContentMessage[] = [];
+
+  loadContentPiecesFromSubscription(
+    subscription,
+    valuesExported,
+    contentPieces,
+  );
+
+  return {
+    id: subscription.id,
+    contentPieces,
+  };
+}
+
+export type ExportedCoValue = {
+  id: string;
+  contentPieces: CojsonInternalTypes.NewContentMessage[];
+};
```

Comment:
Right now exported covalues are opaque, so there's no way to know which schema to load it with, unless you know how it was generated. We should at least keep the CoValue type as part of the `ExportedCoValue` type, and maybe also add some metadata about the CoValue schema?